apps/public/src/app/(general)/contact/actions.ts

```diff
@@ -0,0 +1,34 @@
+'use server'
+
+import { postMeldingByMeldingIdContact } from '@meldingen/api-client'
+import { cookies } from 'next/headers'
+import { redirect } from 'next/navigation'
+
+export const postContactForm = async (_: unknown, formData: FormData) => {
+  const cookieStore = await cookies()
+
+  const meldingId = cookieStore.get('id')
+  const token = cookieStore.get('token')
```

Comment:
I think we should already get `.value` here. If the token exists, but doesn't have a value, we should also early return.